api/src/test/java/org/apache/iceberg/expressions/TestNumericLiteralConversions.java

```diff
@@ -153,17 +154,12 @@ public void testDoubleToDecimalConversion() {
   public void testDecimalToDecimalConversion() {
     Literal<BigDecimal> lit = Literal.of(new BigDecimal("34.11"));
 
-    Assert.assertSame("Should return identical object when converting to same scale",
-        lit, lit.to(Types.DecimalType.of(9, 2)));
-    Assert.assertSame("Should return identical object when converting to same scale",
-        lit, lit.to(Types.DecimalType.of(11, 2)));
-
-    Assert.assertNull("Changing decimal scale is not allowed",
-        lit.to(Types.DecimalType.of(9, 0)));
-    Assert.assertNull("Changing decimal scale is not allowed",
-        lit.to(Types.DecimalType.of(9, 1)));
-    Assert.assertNull("Changing decimal scale is not allowed",
-        lit.to(Types.DecimalType.of(9, 3)));
+    IntStream.range(0, 10).forEach(scale -> {
+      Assert.assertSame("Should return identical object",
+              lit, lit.to(Types.DecimalType.of(9, scale)));
```

Comment:
Nit: indentation is off. It should be 2 indents (4 spaces) for continuation indents.

Reply:
fixed